### DISCUSSION TITLE
Verdadeiro Gamer

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#353] [Imagens]
Post de referência: https://www.vish.blog.br/imagens/tirinhas/verdadeiro-gamer/

**Comment by maryon-telvanni** [comment]
![null](<https://i.postimg.cc/gjJWrkR3/6298173-0077885633-juman.png>)

**Comment by one-eyed-willy** [comment]
> [Maryon Telvanni](</perfil/?user=maryon-telvanni>) disse [sábado, 30 de abril de 2022](<https://www.vish.blog.br/imagens/tirinhas/verdadeiro-gamer/#comment_1113995751>):

> [![null](<https://camo.githubusercontent.com/ab7cd285f4846e399e41bd7b35a7920641fe3af442beb0d01bc30dea69552feb/68747470733a2f2f692e706f7374696d672e63632f676a4a57726b52332f363239383137332d303037373838353633332d6a756d616e2e706e67>)](<https://camo.githubusercontent.com/ab7cd285f4846e399e41bd7b35a7920641fe3af442beb0d01bc30dea69552feb/68747470733a2f2f692e706f7374696d672e63632f676a4a57726b52332f363239383137332d303037373838353633332d6a756d616e2e706e67>)

Foi a primeira coisa que eu pensei também
